### ISSUE TITLE
GMP rational numbers

### ISSUE THREAD

**Issue by bobbbob98** [fsh/integers#1]
Any chance at supporting GMP's rational numbers?

**Comment by fsh** [comment]
Yes, it's something I will eventually need it too, but I cannot give you a practical timeline.
